src/Event/SymfonyBridge.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace PhpAb\Event;
+
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+class SymfonyBridge implements DispatcherInterface
+{
+    /**
+     * @var \Symfony\Component\EventDispatcher\EventDispatcherInterface
+     */
+    private $eventDispatcher;
+
+    /**
+     * @param \Symfony\Component\EventDispatcher\EventDispatcherInterface $eventDispatcher The Symfony EventDispatcher
+     */
+    public function __construct(EventDispatcherInterface $eventDispatcher)
+    {
+        $this->eventDispatcher = $eventDispatcher;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function dispatch($event, $options)
+    {
+        $listeners = $this->eventDispatcher->getListeners($event);
+
+        foreach ($listeners as $listener) {
+            call_user_func($listener, $options);
```

Comment:
Cause symfony forces to dispatch with a second parameter which MUST inherit from the Symfony Event-Class (this is a data-container). It would couple to that class and so i must do this manually. It's also mentioned like this in the blogpost where i found that aproach.